.circleci/config/workflows/website.yml

```diff
@@ -5,3 +5,10 @@ jobs:
         branches:
           only:
             - master
+
+  - algolia_index:
+      context: static-sites
```

Comment:
I don't have access to our Circle contexts but for this particular job to work, we rely on the presence of a sensitive environment variable `ALGOLIA_API_KEY` which I assume will need to be added to this context?

Reply:
sorry is this value stored in that context already? do we just need to get this context added to Nomad instead of adding the value itself?

Reply:
Ah, I was going off of a previous example...I've just added the env var to this context and removed it from the general project

Reply:
I'm assuming since this context is used in other workflows, no additional work is necessary?